dubbo-compatible/src/main/java/com/alibaba/dubbo/rpc/RpcContext.java

```diff
@@ -19,4 +19,33 @@
 
 @Deprecated
 public class RpcContext extends org.apache.dubbo.rpc.RpcContext {
+
+
+    public static RpcContext getContext() {
+        return newInstance(org.apache.dubbo.rpc.RpcContext.getContext());
+    }
+
+    private static RpcContext newInstance(org.apache.dubbo.rpc.RpcContext rpcContext) {
+        RpcContext copy = new RpcContext();
+        copy.getAttachments().putAll(rpcContext.getAttachments());
+        copy.get().putAll(rpcContext.get());
+        copy.setFuture(rpcContext.getFuture());
+        copy.setUrls(rpcContext.getUrls());
+        copy.setUrl(rpcContext.getUrl());
+        copy.setMethodName(rpcContext.getMethodName());
+        copy.setParameterTypes(rpcContext.getParameterTypes());
+        copy.setArguments(rpcContext.getArguments());
+        copy.setLocalAddress(rpcContext.getLocalAddress());
+        copy.setRemoteAddress(rpcContext.getRemoteAddress());
+        copy.setRemoteApplicationName(rpcContext.getRemoteApplicationName());
+        copy.setInvokers(rpcContext.getInvokers());
+        copy.setInvoker(rpcContext.getInvoker());
+        copy.setInvocation(rpcContext.getInvocation());
```

Comment:
`com.alibaba.dubbo.Invocation getInvocation()` can be added to `com.alibaba.dubbo. RpcContext`.

But I think users may rarely use this method directly, so we can consider add this compatible method when necessary.

Reply:
I just make it consistence with org.apache.dubbo.rpc.RpcContext.